core/m3u8lib/parser.py

```diff
@@ -1,11 +1,12 @@
 from urllib.parse import urlparse, urljoin
 from typing import List, Dict
-import requests
 import os
+import requests
 
 
-def fetch_playlist_links(session: requests.Session, playlist_url: str, keep: bool = False) -> List[str]:
-    """ Fetches the m3u8 playlist from the playlist_url
+def fetch_playlist_links(session: requests.Session, playlist_url: str
+                         , keep: bool = False) -> List[str]:
+    """Fetch the m3u8 playlist from the playlist_url.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)